charging_stations_pipelines/db_utils.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import delete, text, update
+from sqlalchemy.orm import Session
+
+from charging_stations_pipelines import settings
+from charging_stations_pipelines.models import address, charging, station
+
+
+def delete_alembic_version(session: Session):
```

Comment:
This seems to be unused. If unused we should delete it.

Reply:
Good catch, it doesn't make sense anyway. If we want to re-run alembic from scratch we have to drop all tables anyway